test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
Any reason this is necessary?  In [previous tests](https://github.com/apigee-127/sway/blob/e53e529dc4a01b11a40f9e968fc15be76a292c73/test/test-issues.js#L69) where we use `./2.0/swagger-relative-refs.yaml` we just do it in that specific test.  Creating a `SwaggerApi` object using `Sway#create` with that file vs. using a helper is trivial so do we really need it?

_(If we decide "No", we could remove the helper completely.)_